README.md

```diff
@@ -573,15 +578,17 @@ The `addInitializer` method is available on the context object that is provided
 - Class element initializers run during class construction, _before_ class fields are initialized.
 - Class _static_ element initializers run during class definition, _before_ static class fields are defined, but _after_ class elements have been defined.
 
+Initializers are called with `this` and the first argument set to the instance of the class for non-static element decorators, and with `this` and the first argument set to the class itself for static element and class decorators.
```

Comment:
Doing both seems problematic to me, and a refactoring hazard. Why not pick one or the other?

Reply:
Fair enough, my thinking was just that it would give more options, but I'm not sure it's worth it to have 2 ways of doing it. I'll update it to just be the argument version, I think that's better overall personally.

Reply:
I’m undecided; i don’t think using arrow functions here is particularly valuable (especially since concise methods are likely to be cleaner) but i definitely think we shouldn’t provide more than one way of doing something.